easybuild/easyconfigs/m/Mako/Mako-1.0.7-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'Mako'
+version = '1.0.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.makotemplates.org'
+description = """A super-fast templating language that borrows the best ideas from the existing templating languages"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae']
+
+download_dep_fail = True
+use_pip = True
+
+builddependencies = [('binutils', '2.31.1')]
+dependencies = [('Python', '3.7.2')]
+
+sanity_check_paths = {
+    'files': ['bin/mako-render'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s-%(version)s-py%(pyshortver)s.egg'],
```

Comment:
This dir is not created, only lib/python../site-p../mako (and Mako-1.0.7.dist-info) when I build it.
Did you check it?

Reply:
On CentOS7, i did get
`.../Compiler/GCCcore/8.2.0/Mako/1.0.7-Python-3.7.2/lib/python3.7/site-packages/Mako-1.0.7-py3.7.egg/mako/*.py`

Reply:
Hmmm, using EB 3.8.1 with or without any changed easyblocks from develop?

Reply:
@akesandgren You're right, I had an outdated customized pythonpackage block.